rmgpy/data/thermo.py

```diff
@@ -1327,6 +1335,23 @@ def estimateRadicalThermoViaHBI(self, molecule, stableThermoEstimator ):
                 logging.error(molecule)
                 logging.error(molecule.toAdjacencyList())
                 raise
+            
+            # Add the radical interactions which are ignored in the thermo calculation of molecule.
+            if saturatedStruct.isAromatic() and (not saturatedStruct.isAtomInCycle(atom)):
+                baseAtom = saturatedStruct.getBaseCb(atom)[0]
```

Comment:
Probably related to the fact that I don't understand the getBaseCb method, but I also don't understand what is going on here. You get the closest Cb to H? Do you need to check all Cbs?